src/_components/card_talk.liquid

```diff
@@ -0,0 +1,20 @@
+<div class="card-talk">
+  {% if show_title %}
+  <div class="card-talk__title">{{ event.title }}</div>
+  {% endif %}
+  <div class="card-talk__speaker">
+    {% if speaker.photo_url %}
+    <figure class="card-talk__profile">
+      <img src="{{ speaker.photo_url }}" class="card-talk__photo" />
+    </figure>
+    {% endif %}
+    <div class="card-talk__info">
+      <div class="card-talk__name text-branded">
+        <span>{{ speaker.first_name }}</span>&nbsp;{{ speaker.last_name }}
+      </div>
+      {% if speaker.twitter_handle %}
+      <div class="card-talk__social">@{{ speaker.twitter_handle }}</div>
```

Comment:
link the twitter handle?

Reply:
Let's revisit this when the speaker pages are added as technically the whole talk card should be clickable and link to the speaker page. So there should not be any either link on the schedule page apart from the link to the speaker page.